tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -572,8 +572,12 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             hostname = volumes[
                 'volume%s.brick%s.hostname' % (index, b_index)
             ]
-            if socket.gethostbyname(NS.node_context.fqdn) != \
-                    socket.gethostbyname(hostname):
+            ip = socket.gethostbyname(hostname)
+            node_id = etcd_utils.read("indexes/ip/%s" % ip).value
+            fqdn = NS.tendrl.objects.ClusterNodeContext(
```

Comment:
before reading fqdn

ensure "node_id" is present at ""indexes/tags/tendrl/integration/:integration_id" , 

otherwise there can be clash between IP address in multi cluster scenario